app/mailers/spree/user_mailer.rb

```diff
@@ -8,10 +8,12 @@ class UserMailer < BaseMailer
     def reset_password_instructions(user, token, _opts = {})
       @edit_password_reset_url = spree.
         edit_spree_user_password_url(reset_password_token: token)
+      subject = "#{Spree::Config[:site_name]} " \
+        "#{I18n.t('devise.mailer.reset_password_instructions.subject')}"
```

Comment:
I did a little investigation on the en.yml file and I found that we already have entries under
spree.user_mailer.reset_password_instructions (line 3563 of en.yml)
these are used in the email body:
app/views/spree/user_mailer/reset_password_instructions.text.erb

I think the translation of the subject should also go there.
I think this should be:
`I18n.t('spree.user_mailer.reset_password_instructions.subject`
and then the en.yml entry for subject: would be at line 3573 just after the issue_text entry.

Reply:
Thanks for the reference Louis. Fixing the translation. :slightly_smiling_face: